internal/build/src/tasks/full-bundle.ts

```diff
@@ -1,6 +1,7 @@
 import path from 'path'
 import { nodeResolve } from '@rollup/plugin-node-resolve'
 import { rollup } from 'rollup'
+import replace from '@rollup/plugin-replace'
```

Comment:
It seems the rollup replace plugin is a ghost dependency. It may be better to add it to the package.json.